### PR TITLE
Add language to title on Quickstart, Add to App, RBAC & Hierarchies pages

### PR DESCRIPTION
**Pull request by dhatch**
- Adds a new `metaTitle` param to page front matter that will substitute LANG
  for the current pages language, allowing us to customize the title used by
  search to include the language.
- Adds to H1 the current language on some pages that we want to appear for SEO.

### CODE REVIEW

File: docs/content/any/getting-started/application/enforce/index.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 ---
 title: Enforce authorization
+metaTitle: Enforce authorization in LANG
```
Comment (samscott89) [inline docs/content/any/getting-started/application/enforce/index.md:3]: Maybe `$LANG` to make it more obvious it's interpolated?

File: docs/themes/oso-tailwind/layouts/partials/head.html [hunk 2]
```diff
@@ -2,11 +2,16 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>{{ block "title" . }}
+        {{- if .Params.metaTitle -}}
+        {{- replace .Params.metaTitle "LANG" .Page.Language.LanguageName }} - {{ .Site.Title -}}
+        {{- else -}}
         {{- if .Title -}}
```
Comment (samscott89) [inline docs/themes/oso-tailwind/layouts/partials/head.html:8]: Make this an `else if`

**Review by samscott89 — COMMENTED** [review]
Looking at the source for the docs, we also have twitter + open graph titles (used when creating link snippets). I don't think those will currently be updated based on the change. But that might not matter

File: docs/themes/oso-tailwind/layouts/partials/head.html [hunk 3]
```diff
@@ -2,11 +2,16 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>{{ block "title" . }}
+        {{- if .Params.metaTitle -}}
+        {{- replace .Params.metaTitle "LANG" .Page.Language.LanguageName }} - {{ .Site.Title -}}
```
Comment (samscott89) [inline docs/themes/oso-tailwind/layouts/partials/head.html:6]: I wonder whether we actually want the end of this " - Oso Documentation". Is this good for SEO?

Comment (samscott89) [reply docs/themes/oso-tailwind/layouts/partials/head.html:6]: (Same for non-meta title changes)

Comment (dhatch) [reply docs/themes/oso-tailwind/layouts/partials/head.html:6]: I think it's fine but I'll check with Stephie

**Comment by dhatch** [comment]
> Looking at the source for the docs, we also have twitter + open graph titles (used when creating link snippets). I don't think those will currently be updated based on the change. But that might not matter

Yeah no way to do those as far as I can tell unless we vendor those templates & add the string replacement to them.

**Review by samscott89 — APPROVED** [review]

**Pull request merged**